Dockerfile

```diff
@@ -3,4 +3,4 @@ ADD build/libs/payments.jar /srv/payments.jar
 WORKDIR /srv
 EXPOSE 8080
 RUN mkdir config
-ENTRYPOINT ["java","-Dspring.profiles.active=container","-jar","/srv/payments.jar"]
+ENTRYPOINT ["java","-Dspring.profiles.active=production","-jar","/srv/payments.jar"]
```

Comment:
For example using the `SPRING_PROFILES_ACTIVE` environment variable (which you do now).